lib/coney/consumer_executor.ex

```diff
@@ -9,7 +9,7 @@ defmodule Coney.ConsumerExecutor do
   rescue
     exception ->
       if function_exported?(consumer, :error_happened, 3) do
-        exception
+        {exception, System.stacktrace()}
```

Comment:
Also I think, maybe let's do it in backwards-compatible manner? then I will bump only `minor` version

```elixir
cond do
  function_exported?(consumer, :error_happened, 3) ->
    exception
    |> consumer.error_happened(payload, meta)
    |> handle_result(task)
  function_exported?(consumer, :error_happened, 4) ->
    exception
    |> consumer.error_happened(System.stacktrace(), payload, meta)
    |> handle_result(task)
  true ->
    reject(task)
end
```

what do you think?

Reply:
Perfect.